sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -1088,10 +1088,12 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
       programYearTotalProvincialFSC: 0,
       programYearTotalScholarshipsBursaries: 0,
       programYearTotalSpouseContributionWeeks: 0,
+      programYearTotalReturnTransportation: 0,
```

Comment:
Should it be `programYearTotalReturnTransportationCost` to be aligned with the others?